src/Text/Pandoc/Writers/LaTeX.hs

```diff
@@ -1494,20 +1494,14 @@ citationsToBiblatex (c:cs)
 
       return $ text cmd <> (mconcat groups)
 
-  where grouper prev cit = grouper' (citationPrefix cit)
-               (citationSuffix cit) (citationId cit) 
-            where grouper' [] sfx cid 
-                     | null sfx = addToGroup id 
-                     | otherwise = addToGroup (\(CiteGroup pfx _ ids) -> 
-                            CiteGroup pfx sfx ids) 
-                     where addToGroup fn = case prev of  
-                             ((CiteGroup pfx oSfx ids):rest) 
-                                 | null oSfx ->
-                                     (fn (CiteGroup pfx oSfx (cid:ids))):rest
-                                 | otherwise -> (fn (CiteGroup pfx [] [cid])):prev
-                             [] -> (fn (CiteGroup [] [] [cid])):prev
-                  grouper' pfx [] cid = (CiteGroup pfx [] [cid]):prev
-                  grouper' pfx sfx cid = (CiteGroup pfx sfx [cid]):prev 
+  where grouper prev cit = case prev of  
+         ((CiteGroup oPfx oSfx ids):rest)
+             | null oSfx && null pfx -> (CiteGroup oPfx sfx (cid:ids)):rest
+             | null pfx -> (CiteGroup oPfx sfx [cid]):prev 
```

Comment:
Much clearer!  I understand line 1499:
```
(prefix1 cite1 [no suffix]) ([no prefix] cite2 suffix2) ==> prefix1 [citet,cite2] suffix2
```
But I'm not sure I get 1500.  Doesn't it amount to this?
```
(prefix1 cite1 suffix1) ([no prefix] cite2 suffix2) ==> prefix1 cite1 suffix1 ; prefix1 cite1 suffix2
```
which would mean prefix1 (oPfx) gets doubled? Maybe I'm missing something.

Reply:
that's a new suffix but no new prefix, the new suffix ends the group and starts with that same prefix.

Reply:
Hm, still not sure I get it.  Do you have a test case like
```
[pref1 @cite1 suff1; @cite2 suff2]
```
and what does it spit out?

Reply:
Yes we have:
 ``% pandoc -t latex --biblatex
[e.g. @a1, blah;@a2, ch.3 and elsewhere;@a3; but also @b4;@b5]``
 which spits out:
``\autocites[e.g.~][blah]{a1}[e.g.~][ch.3 and
elsewhere]{a2}[e.g.~][]{a3}[but also][]{b4,b5}``

Reply:
I see: so in this case the `e.g.~` gets repeated for a1, a2, and a3. Is that expected behavior?
I would have expected just one `e.g.~` at the beginning if I were writing this markdown.

Reply:
So would you want something like 
``\autocites[e.g.~][blah]{a1}[ch.3 and elsewhere]{a2}[]{a3}[but also][]{b4,b5}``?

Reply:
Yes, that's what I'd expect.  I suppose you could check back with the author of the original issue, but I can't really imagine anyone wanting the `e.g.` to be repeated like that.
And imagine something like
```
[in order of importance, @a1, p. 34; @a2, p. 44]
```
Here it would be quite bad if this were rendered "in order of importance, Smith 2000, p. 34, in order of importance, Jones 1999, p. 44".

Reply:
is this output correct:
\autocites[e.g.~][blah]{a1}[ch.3 and elsewhere]{a2}{a3}[but also][]{b4,b5}
does it need another bracket between {a2} and {a3}?

Reply:
I tried it.  With a numerical style it comes out as

> [e.g. 1, blah, 2, ch.3 and elsewhere, 3, but also 4, 5]

With author-year

> (e.g. Doe 2005, blah; Doe 2006, ch.3 and elsewhere; Doe 2007; but also Doe 2008; Doe 2009)

This looks right to me!